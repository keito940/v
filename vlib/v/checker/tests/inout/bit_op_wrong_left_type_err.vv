fn main() {
	0.5 & 1
}