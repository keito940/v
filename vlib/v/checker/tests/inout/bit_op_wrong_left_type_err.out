vlib/v/checker/tests/inout/bit_op_wrong_left_type_err.v:2:6: error: operator & not defined on left type `f64`
    1| fn main() {
    2|     0.5 & 1
               ^
    3| }
